docl/commands.py

```diff
@@ -216,14 +222,18 @@ def run(mount=False, label=None, details_path=None, tag=None):
                                                 label=label,
                                                 details_path=details_path)
     _ssh_setup(container_id, container_ip)
+    _set_remote_variables(container_id, container_ip)
+
+
+def _set_remote_variables(container_id, container_ip):
```

Comment:
not sure `_set_remove_variables` is the correct name. This function calls `update_running_system.py`